### ISSUE TITLE
SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing else if behavior

### ISSUE THREAD

**Issue by Ekimik** [slevomat/coding-standard#1563]
Hi,
I don't know if I should call it false positive error or not, but consider following code
```php
<?php

namespace Ns;

class Foo {

	public function foo(int $foo): string {
		$name = 'foobarbaz';

		if ($foo === 1) {
			$name = 'foo';
		} else if ($foo === 2) {
			$name = 'bar';
		} else if ($foo === 3) {
			$name = 'baz';
		}

		return $name;
	}

}
```

and following rule
```xml
<rule ref="SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing">
	<properties>
		<property name="linesCountBefore" value="1" />
		<property name="linesCountAfter" value="1" />
	</properties>
</rule>
```

Following errors are found
```
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 12 | ERROR   | [x] Expected 1 line before "if", found -1.
    |         |     (SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing.IncorrectLinesCountBeforeControlStructure)
 14 | ERROR   | [x] Expected 1 line before "if", found -1.
    |         |     (SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing.IncorrectLinesCountBeforeControlStructure)
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``` 

We are using `else if` syntax in our code now. When I use `elseif` syntax, there are no errors. We are moving to `elseif` syntax now, so it's not really a problem for us, but I think it's good to report it.

**Comment by kukulich** [comment]
Thanks, but yes, this is expected and won't be fixed :)
